docs/settings/i18n-settings.asciidoc

```diff
@@ -0,0 +1,20 @@
+[role="xpack"]
+[[i18n-settings-kb]]
+=== i18n settings in Kibana
+++++
+<titleabbrev>i18n Settings</titleabbrev>
+++++
```

Comment:
```suggestion

```
Please remove the <titleabbrev>, this will break pages once we migrate to the new Asciidoctor build process.